src/cdk-experimental/testing/component-harness.ts

```diff
@@ -252,3 +258,78 @@ export interface ComponentHarnessConstructor<T extends ComponentHarness> {
    */
   hostSelector: string;
 }
+
+/** A class used to represent a ComponentHarness class with additional predicates applied. */
```

Comment:
Could you expand this description block to give broader context for how this class is used? It also seems off to me that it says "represent[s] a ComponentHarness", but it seems like it's only _associated_ with a harness, no?